_posts/2022-04-25-tardigrade-launch.md

```diff
@@ -0,0 +1,301 @@
+---
+layout: post
+title:  "ETL in Trino speeds with project Tardigrade: now ready for early users"
+author: Andrii Rosa, Brian Zhan, Lukasz Osipiuk, Zebing Lin
+excerpt_separator: <!--more-->
+---
+
+After 6 months of challenging work, we’ve launched Project Tardigrade. It is 
+designed to improve user experience of running resource intensive queries that
+are common in ETL space. It’s available right now! It required some significant
+and fascinating engineering to get us here. Read on to learn how Project 
+Tardigrade works and how to enable it. We think you’ll enjoy hearing about it.
+
+<!--more-->
+
+## What is Project Tardigrade
+
+What we love most about doing ETL with Trino is that you get fast query speeds,
+ and you can iterate fast with intuitive error messages, interactive 
+ experience, and query federation.
+
+One of the big problems that had bugged us for a long time is that doing 
+long-running ETL workload with Trino is hard. You've probably heard of the 
+below issues:
+
+* Reliable landing times: Queries that run for hours can fail and restart 
+  from scratch. This wastes CPU resources and makes it hard for you to meet 
+  your data landing SLA. 
+* Cost-efficient clusters: Trino queries that need terabytes of distributed 
+  memory require extremely large clusters today due to lack of iterative 
+  execution.
+* Concurrency: Multiple independent clients may submit their queries 
+  concurrently. Due to the lack of available resources at a certain moment some
+  of these queries may need to be killed and restarted from zero after a 
+  while. This makes the landing time hardly predictable.
+
+While it is certainly possible to 
```

Comment:
Reword.. too much hedging .. just be definitive .. It can be done with a large team of experts .. but that is not accessible to most Trino users..